### DISCUSSION TITLE
Alpha 2 Version

### DISCUSSION THREAD

**Discussion by ferdiworks** [rrirower/DP2SceneGraphWizard#4] [General]
I downloaded the zip file

Are you going to update the installer file .exe for alpha 2 ?

I can still test it without the installer!

**Comment by ferdiworks** [comment]
OK Got it Thanks

**Reply by rrirower** [reply]
Make sure to re-run the wizard as it will download the latest zip file.  I had to make several changes to the zip.

**Comment by ferdiworks** [comment]
Worked well!
I created a channel and got it packaged and ran mytvshow / Static Analysis

Severity | Definition
-- | --
Error | Channel issue that will block channel from passing certification.
Warning | Channel issue that does not block certification, but may be reported as an Error in the future.
Info | Helpful information about your channel detected during testing.

#	Message	Severity	Category	Certification requirements

1	Channels are required to support roInput events. Please ensure your manifest includes the supports_input_launch=1 entry and your channel properly handles roInput events.	Error	DeepLinking	[5.2](https://developer.roku.com/docs/developer-program/certification/certification.md#certification-criteria-1)

2	If your channel has streamed more than an average of 5 million hours per month over the last three months or you have a new channel projected to reach the specified streaming hour threshold shortly after launch, you must participate in Roku’s Continue Watching program. See Continue Watching for more detail. TVOD, live linear, and made-for-kids channels are excluded from this requirement.	Warning	Uncategorized	[4.13](https://developer.roku.com/docs/developer-program/certification/certification.md#certification-criteria-1)

3	Channel version wasn't updated.	Warning	Manifest	[6.1](https://developer.roku.com/docs/developer-program/certification/certification.md#certification-criteria-1)

4	For your channel to pass certification, your application must fire the "AppDialogInitiate" and "AppDialogComplete" beacons if the channel UI displays a login, user selection, EULA, or any other dialog before the home page.	Warning	Performance	[3.2](https://developer.roku.com/docs/developer-program/certification/certification.md#certification-criteria-1)

Only Error was deep linking! =     Error	DeepLinking

Channel version wasn't updated. in the manifest! I could of done this manually but did not!

**Reply by rrirower** [reply]
Yes, there's still a few things on my list to finish. I'll hopefully have more of them done in a week or two.

**Reply by ferdiworks** [reply]
I made a fast video to demonstrate how the software works
https://c.themediacdn.com/embed/media/W3s3aS/iTeSM39sAwz/O4FPUCsMeAY_1?preview=1

**Reply by rrirower** [reply]
Be advised, the wizard software is not complete yet. I am working on updates for it as well as updates for the BrightScript code.

Fyi... The wizard software now allows the user to enter the version number and saves all fields entered by the user so that they can be loaded again.

These changes will be made available in Alpha-3.

Thanks again for your support.

**Reply by ferdiworks** [reply]
Thats awesome! But the question is do we really need to save the data as each DP channel we are converting would use a different feed links, diffeerent version numbers and images etc.

**Reply by rrirower** [reply]
Saving the data allows the user to rerun the wizard tool in the future without having to re-enter all of the info.  For example, if the user changes images or version numbers he won't have to re-enter all of the remaining data.

**Comment by ferdiworks** [comment]
Final testing of Alpha 2 is done on my end. Does what is suppose to do, Looking forward to test Alpha 3

**Reply by ferdiworks** [reply]
I made another Donation today to help the project!

**Reply by rrirower** [reply]
Again, thanks for your support.

Alpha-3 will be pushed soon.  Working on partial support of deep linking.

**Reply by ferdiworks** [reply]
I was able to take the Deep Linking template from the Roku sample master and edit it with your code and get it to parse the feed and categories and package it and upload it to roku for test and it past all test. BUT it was made for Subscrption  and Ads so my free channel got warnings only for the Ads and subscription stuff!

**Reply by rrirower** [reply]
Be advised: I am in the process of separating out the Subscription and Ad requirements from deep linking.  I found that the template examples are too heavily interconnected with the rest of the code.  I have deep linking working for some content types.  You can continue down your own coding path, or, wait for the next alpha release.

**Reply by ferdiworks** [reply]
I am waiting for your next release. I dont need all that other stuff. I just was tesing it and wanted to see if it worked with some edits to the code! I am supporting ur product as I liked what you done so far and will continue to test it with the next release!
